ci/infra/testrunner/tests/test_masters.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+
+def test_remove_master(deployment, conf, platform, skuba, kubectl):
```

Comment:
I think this test shouldn't use the `deployment` fixture which joins all nodes, but `bootstrap` and then join all but one master and all workers. It is just a couple of lines of code and doesn't need to change the behavior of the `deployment` fixture.

Reply:
I switched this to be a nightly test so if we remove one of the nodes it won't affect other tests